### PR TITLE
fix port usage misunderstanding

### LINKED ISSUE #373: port confusion, document default port documentation in docker image
hi,
the default ports for the metrics in the binary is 80 and 81 while the readme says the default for the main metrics is 8080 in the first paragraph and the telemetry port default is not mentioned at all.
To make it worse the docker image sets one of them by entrypoint to 8080 but leaves the telemetry port at the privileged 81.

This is pretty confusing, the readme should not state that the default is 8080 when this it really is 80 in the binary, the documentation should probably mention that the docker image changes the default and the docker image should probably also change the default of the telemetry port so that it does not default to a privileged port (or not change the default at all).

### PR DESCRIPTION
**Pull request by andyxning**
Fix #373 

/cc @brancz 
ping @juliantaylor

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/375)

### CODE REVIEW

File: kubernetes/kube-state-metrics-deployment.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ spec:
         ports:
         - name: http-metrics
           containerPort: 8080
+        - name: telemetry-metrics
```
Comment (brancz) [inline kubernetes/kube-state-metrics-deployment.yaml]: From the build failure:

> The Deployment "kube-state-metrics" is invalid: spec.template.spec.containers[0].ports[1].name: Invalid value: "telemetry-metrics": must be no more than 15 characters

I'd suggest `self-metrics`.

Comment (andyxning) [reply kubernetes/kube-state-metrics-deployment.yaml]: I have changed the port name to `telemetry`. :)

**Comment by brancz** [comment]
lgtm

**Comment by juliantaylor** [comment]
thanks

**Pull request merged**